mlrun/serving/server.py

```diff
@@ -405,6 +411,94 @@ def v2_serving_init(context, namespace=None):
     _set_callbacks(server, context)
 
 
+async def async_execute_graph(
+    context: MLClientCtx,
+    data: DataItem,
+    batching: bool,
+    batch_size: Optional[int],
+) -> (list[Any], Any):
```

Comment:
I'm thinking we should probably remove the termination result, because it's never been documented or tested via mlrun, and managing termination result over a split graph is also something I'm not sure we can currently due from mlrun.

Basically, it's always been a feature of storey that's used only for testing.

Reply:
By split graph, I mean nonlinear, not multiple functions.

Reply:
ok, so let's discuss this. Returning the termination result was something that was brought up in the design of this feature as something that should be exposed. If there's nobody using it at all and it's not in use in MLRun (vs. directly with storey) then I agree this can be removed.

Reply:
Removed termination result.